lambda_functions/ml_service/template.yaml

```diff
@@ -51,6 +51,7 @@ Resources:
         - arn:aws:iam::aws:policy/ComprehendFullAccess
         - arn:aws:iam::aws:policy/AmazonS3FullAccess
         - arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
```

Comment:
Can this be removed now that Sagemaker is moved behind the API Gateway?

Reply:
Yes, Sagemaker access is not required anymore